src/lib/containers/QueryWrapper.tsx

```diff
@@ -339,7 +339,7 @@ export default class QueryWrapper extends React.Component<
       },
     )
 
-    const loadingCusrorClass = isLoading ? 'SRC-logo-cursor' : ''
+    const loadingCusrorClass = isLoading ? 'SRC-logo-cursor' : 'SRC-wrapper'
```

Comment:
I think the class 'SRC-wrapper' can be fixed to the top move div, it doesn't need to be conditional.

Reply:
isn't this the topmost div?